### ISSUE TITLE
Documentation: code style

### ISSUE THREAD

**Issue by KulaginVladimir** [festim-dev/FESTIM#852]
The FESTIM code is formatted with [Black](https://github.com/psf/black).

I think it'd be beneficial to explicitly specify this in the [Contributing to the code](https://festim.readthedocs.io/en/latest/devguide/index.html#contributing-to-the-code) section of developer's guide.

**Comment by RemDelaporteMathurin** [comment]
It would also help to show a few commands on how to format a script with the black command 
```
black my_script.py
```

or 

```
black .
```

**Comment by XinShen-CHN** [comment]
I found it that the content of the index.rst under devguide is different with the content of the merged ones in Developer's guide in FESTIM documentation when I want to make the modification to specify the code style.

Here's the content of the index.rst under devguide:
```
1) `Fork the repository <https://github.com/festim-dev/FESTIM/fork>`_

By forking the repository, you create a copy where you can safely make changes.

2) Make your changes
3) `Open a PR <https://github.com/festim-dev/FESTIM/compare>`_
4) Wait for a :ref:`maintainer<Maintainers>` to review your PR

Before merging your changes, they have to be reviewed. We ensure the changes don't break anything during the review and eventually propose/request improvements.
The time before the review will depend on the maintainers' availability.

5) When everything is in order, the maintainers will merge your PR!
```

Here is the content of the merged in Developer's guide in FESTIM documentation: [https://festim.readthedocs.io/en/latest/devguide/index.html](https://festim.readthedocs.io/en/latest/devguide/index.html).

**Comment by RemDelaporteMathurin** [comment]
This is the difference between the `latest` and `stable` versions of the documentation.

`stable` is the latest released festim version (here 1.3.1)
`latest` is the latest pushed version (on the main branch)
